### PR TITLE
feat(web, subgraph): changes in terms form ipfs upload, add user entity in subgraph, fetch actual escrows

### PR DESCRIPTION
**Pull request by kemuru**

### CODE REVIEW

**Review by jaybuidl — APPROVED** [review]

File: subgraph/mappings/utils.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { BigInt } from '@graphprotocol/graph-ts'
+
+export const ZERO = BigInt.fromI32(0)
```
Comment (alcercu) [inline subgraph/mappings/utils.ts:3]: I think there exists a `BigInt.zero` built in 🤔

File: web/src/utils/handleFileUpload.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      if (deliverableFile.type !== "application/pdf") {
+        alert("That type of file is not valid. Please upload a PDF file.");
+        setIsFileUploading(false);
+        return;
+      }
+
+      const fileResponse = await uploadFileToIPFS(deliverableFile);
+      const fileData = await fileResponse.json();
+      const fileHash = fileData.cids[0];
+      transactionDetails.extraDescriptionUri = fileHash;
```
Comment (alcercu) [inline web/src/utils/handleFileUpload.ts:29]: There was some discussion regarding using `ipfs://...` to conform with RFC3986, we will need to update `uploadFileToIpfs` to make it return the cids with that format.

**Review by alcercu — CHANGES_REQUESTED** [review]
There are a couple of `console.log` too, let's remember to remove them before announcing testnet.

**Comment by kemuru** [comment]
> There are a couple of `console.log` too, let's remember to remove them before announcing testnet.

yep, I'm still using some for testing, will delete them before release

**Pull request merged**